gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallableFactory.java

```diff
@@ -224,6 +230,9 @@ private GrpcCallableFactory() {}
           new GrpcServerStreamingRequestParamCallable<>(
               callable, grpcCallSettings.getParamsExtractor());
     }
+    callable =
+        new GrpcExceptionServerStreamingCallable<>(callable, ImmutableSet.<StatusCode.Code>of());
```

Comment:
I think you need to pass through `callSettings.getRetryableCodes()` here, otherwise no retries will be done. (This appears to not be relevant yet, without a retry implementation, but it will be wrong when the retry implementation is added)

Reply:
`callSettings.getRetryableCodes()` doesn't currently exist for ServerStreamingCallSettings. It will be introduced in the retry PR. This PR just sets the scene for it